mobile-browser-based-version/src/core/communication/client_builder.ts

```diff
@@ -1,13 +1,13 @@
-import { DecentralisedClient } from './decentralised/decentralised_client'
-import { FederatedClient } from './federated/federated_client'
 import { Client } from './client'
-import { Task } from '../task_definition/base/task'
+import { DecentralizedClient } from './decentralized/decentralized_client'
+import { FederatedClient } from './federated/federated_client'
 import { Platform } from '../../platforms/platform'
+import { Task } from '../task/base/task'
 
 export function getClient (platform: Platform, task: Task, password: string): Client {
   switch (platform) {
     case Platform.decentralized:
-      return new DecentralisedClient(
+      return new DecentralizedClient(
```

Comment:
we need to decide if to use the British or the US convention ^^

Reply:
US